assignments/closure.js

```diff
@@ -1,7 +1,11 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const username = 'Heorhii Siburov'
+  function greetMe() {
+    console.log('Hi there, ' + username + '!')
+  }
 
```

Comment:
Remember closures are functions which have access to variables declared outside of their scope.